run-dev-tmux.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+tmux_new_window() {
+    local SESSION=${1}
+    local ID=${2}
+    local CMD=${3}
+    local NAME="${SESSION}-${ID}"
+
+    tmux new-window -t "${SESSION}:${ID}" -n "${NAME}"
+    tmux send-keys -t "${NAME}" "${CMD}" C-m
+}
+
+build_system_contracts() {
+    local CMD=(
+        "make -s"
+        "build-contracts-rs"
+        "CARGO_FLAGS=--quiet"
+    )
+
+    echo "Building system contracts..."
+    ${CMD[*]}
+}
+
+build_node() {
+    local CMD=(
+        "cargo build"
+        "--quiet"
+        "--manifest-path=node/Cargo.toml"
+    )
+
+    echo "Building node..."
+    ${CMD[*]}
+}
+
+generate_timestamp() {
+    local SCRIPT=(
+        "from datetime import datetime, timedelta;"
+        "print((datetime.utcnow() + timedelta(seconds=40)).isoformat('T') + 'Z')"
```

Comment:
My antique computer needs this to be configurable, sadly.

Reply:
As I write this I recognize that the onus is on me to find a fix, not you of course.